### DISCUSSION TITLE
Adding hcpasl pipeline

### DISCUSSION THREAD

**Discussion by lucsanara** [neurodesk/neurodesk.github.io-hugo#402] [New Tool]
Hello!!
Could you please add hcpasl pipeline to neurodesk? We want to try processing asl data with that tool. You may find more information [here](https://github.com/physimals/hcp-asl). 
Thank you in advance!

**Comment by stebo85** [comment]
That could be possible, but we will need your help in adding and testing it. Can you send a pull request that adds it to the neurocontainers repository? 

Some examples that are close to start from:
https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/connectomeworkbench/build.sh
https://github.com/NeuroDesk/neurocontainers/blob/0259f7d74df0ca80a9f735b2bae58b05606ff724/recipes/hdbet/build.sh

**Comment by stebo85** [comment]
this is done.
